common/changes/ianc-removing-deprectated-tslint-rules_2016-12-22-19-50.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/gulp-core-build-typescript",
+      "comment": "Fixing TSLint task by removing some deprecated rules (\"label-undefined”, \"no-duplicate-key\", and \"no-unreachable\") and setting the \"noUnusedParameters\" and \"noUnusedLocals\" TS compiler options to cover the deprecated \"no-unused-variable\".",
+      "type": "patch"
```

Comment:
The build is still broken though but I will approve assuming you'll fix the build break